app/components/paperShow/refCitedPapers/mobileRefCitedPapers.tsx

```diff
@@ -32,23 +32,26 @@ const MobileRefCitedPapers: FC<Props> = ({ type, parentPaperId, paperCount, hist
   const isLoading = useSelector((state: AppState) => {
     return type === 'reference' ? state.paperShow.isLoadingReferencePapers : state.paperShow.isLoadingCitedPapers;
   });
-
   const [query, setQuery] = useState<string>('');
   const [sortOption, setSortOption] = useState<PAPER_LIST_SORT_TYPES>('NEWEST_FIRST');
-
-  useEffect(
-    () => {
-      const queryParamsObject = getQueryParamsObject(location.search);
-      if (type === 'reference') {
-        setSortOption(queryParamsObject['ref-sort'] || 'NEWEST_FIRST');
-        setQuery(queryParamsObject['ref-query'] || '');
-      } else if (type === 'cited') {
-        setSortOption(queryParamsObject['cited-sort'] || 'NEWEST_FIRST');
-        setQuery(queryParamsObject['cited-query'] || '');
-      }
-    },
-    [location.search, type]
-  );
+  const wrapperNode = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    const queryParamsObject = getQueryParamsObject(location.search);
+    if (type === 'reference') {
+      setSortOption(queryParamsObject['ref-sort'] || 'NEWEST_FIRST');
+      setQuery(queryParamsObject['ref-query'] || '');
+    } else if (type === 'cited') {
+      setSortOption(queryParamsObject['cited-sort'] || 'NEWEST_FIRST');
+      setQuery(queryParamsObject['cited-query'] || '');
+    }
+  }, [location.search, type]);
+
+  useEffect(() => {
+    if (isLoading && location.state?.scrollTo === type && wrapperNode.current) {
+      window.scrollTo(0, wrapperNode.current.offsetTop - 100);
```

Comment:
It will be easier to understand if you use comments to describe `100` or define it as a global variable.